subsystem/interlocks.py

```diff
@@ -1,60 +1,157 @@
 # interlocks.py
 import tkinter as tk
 import os, sys
+import instrumentctl.g9_driver as g9_driv
+from utils import LogLevel
+import time
 
+# the bit poistion for each interlock
+INPUTS = {
+    0 : "E-STOP Int", # Chassis Estop
+    1 : "E-STOP Int", # Chassis Estop
+    2 : "E-STOP Ext", # Peripheral Estop
+    3 : "E-STOP Ext", # Peripheral Estop
+    4 : "Door", # Door 
+    5 : "Door", # Door Lock
+    6 : "Vacuum Power", # Vacuum Power
+    7 : "Vacuum Pressure", # Vacuum Pressure
+    8 : "High Oil", # Oil High
+    9 : "Low Oil", # Oil Low
+    10 : "Water", # Water
+    11 : "HVolt ON", # HVolt ON
+    12 : "G9SP Active" # G9SP Active
+    }
+
+def handle_errors(self, data):
+    try:
+        response = g9_driv.read_response()
+        return {"status":"passes", "message":"No errors thrown at this time."}
+
+    except ValueError as e:
+        return {"status":"error", "message":str(e)}
```

Comment:
This is being defined outside the class (`self` makes no sense as a param) and not used. I think this should be deleted and implement error handling directly in `update_data`